src/modules/did/did.service.ts

```diff
@@ -338,7 +358,7 @@ export class DIDService implements OnModuleInit {
       });
       // Only refreshing a DID that is already cached.
       // Otherwise, cache could grow too large with DID Docs that aren't relevant to Switchboard
-      if (didDocEntity) {
+      if (didDocEntity !== null) {
```

Comment:
I just wanted to be more specific suspecting that this is the cause of problem I had with testing. `didDocEntity` can not be `undefined`, because `findById` returns `DIDDocumentEntity` or `null`. But test was failing for another reason, so it is not necessary to complicate this check. I'll revert it